app/uk/gov/hmrc/agentsubscriptionfrontend/views/update_business_address.scala.html

```diff
@@ -0,0 +1,84 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.agentsubscriptionfrontend.config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.{form, input, errorSummary}
+@import uk.gov.hmrc.agentsubscriptionfrontend.controllers.routes
+@import uk.gov.hmrc.agentsubscriptionfrontend.models.UpdateBusinessAddressForm
+@(addressForm: Form[UpdateBusinessAddressForm])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@uk.gov.hmrc.agentsubscriptionfrontend.views.html.main_template(appConfig, title =  error_prefix(addressForm) + Messages("updateBusinessAddress.title"), bodyClasses = None) {
+    @if(addressForm.hasErrors) {
+        @errorSummary(Messages("error.summary.heading"), addressForm, Seq.empty, Some("update-business-address"))
+    }
+    <h1 class="heading-xlarge">@Messages("updateBusinessAddress.title")</h1>
+
+    <p>@Messages("updateBusinessAddress.p1")</p>
+
+    <p class="panel panel-border-wide">@Messages("updateBusinessAddress.p2")</p>
+
+    @form(action = routes.BusinessIdentificationController.submitUpdateBusinessAddressForm(), 'class -> "form js-form") {
+
+        <fieldset>
+
+            <div class="form-group">
```

Comment:
lets put this in a for loop that outputs 
            <div class="form-group">
            @input(
                addressForm("addressLine1"),
                '_label -> Messages("updateBusinessAddress.address_line_1.title"),
                '_inputClass -> "form-control input--cleared",
                '_labelClass -> "heading-small soft--ends"
            )
            </div>
then postcode